app/src/main/AndroidManifest.xml

```diff
@@ -47,6 +47,16 @@
         android:name="android.permission.QUERY_ALL_PACKAGES"
         tools:ignore="QueryAllPackagesPermission" />
 
+    <!-- Permission used to open settings -->
+    <permission 
+        android:name="com.stevesoltys.seedvault.opensettings"
```

Comment:
I would suggest changing this to "com.stevesoltys.seedvault.OPEN_SETTINGS", following AOSP permission naming